Libplanet.Action/ActionEvaluator.cs

```diff
@@ -98,6 +98,16 @@ public class ActionEvaluator : IActionEvaluator
             IPreEvaluationBlock block,
             HashDigest<SHA256>? baseStateRootHash)
         {
+            if (block.ProtocolVersion < BlockMetadata.PBFTProtocolVersion)
+            {
+                throw new BlockProtocolVersionNotSupportedException(
+                    $"The native implementation does not support an evaluation of a block " +
+                    $"#{block.Index} pre-evaluation hash {block.PreEvaluationHash} " +
+                    $"with protocol version less than {BlockMetadata.PBFTProtocolVersion}: " +
+                    $"{block.ProtocolVersion}",
+                    block.ProtocolVersion);
+            }
+
```

Comment:
Is the removal of old behavior in another PR?

Reply:
This only prevents evaluating PoW blocks. Unnecessary code will be removed in a separate PR.